apps/system/js/app_window.js

```diff
@@ -1703,6 +1703,12 @@
 
   AppWindow.prototype.getIconForSplash =
     function aw_getIconForSplash(manifest) {
+      if (this.isPrivate) {
+        this._splash = '/style/icons/pb_icon.png';
+        this.preloadSplash();
+        return '/style/icons/pb_icon.png';
```

Comment:
nit: ```return this._splash;``` so I don't have to change this path twice in the future! :-p

I would also opt to call this private_browser_splash.png or similar so it's more readable.